blueprints/app/files/app/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
I think it's the most common, and having it there will prompt users to set it for themselves if it's different for their app.